src/main/java/bio/terra/cli/command/config/getvalue/Logging.java

```diff
@@ -1,28 +1,56 @@
 package bio.terra.cli.command.config.getvalue;
 
+import bio.terra.cli.command.helperclasses.BaseCommand;
+import bio.terra.cli.command.helperclasses.FormatOption;
 import bio.terra.cli.context.GlobalContext;
-import java.util.concurrent.Callable;
-import org.slf4j.LoggerFactory;
+import bio.terra.cli.context.utils.Logger;
+import picocli.CommandLine;
 import picocli.CommandLine.Command;
 
 /** This class corresponds to the fourth-level "terra config get-value logging" command. */
 @Command(name = "logging", description = "Get the logging level.")
-public class Logging implements Callable<Integer> {
-  private static final org.slf4j.Logger logger = LoggerFactory.getLogger(Logging.class);
+public class Logging extends BaseCommand {
+  @CommandLine.Mixin FormatOption formatOption;
 
+  /** Return the logging level properties of the global context. */
   @Override
-  public Integer call() {
-    GlobalContext globalContext = GlobalContext.readFromFile();
+  protected void execute() {
+    LoggingReturnValue loggingLevels =
+        new LoggingReturnValue(globalContext.consoleLoggingLevel, globalContext.fileLoggingLevel);
+    formatOption.printReturnValue(loggingLevels, Logging::printText);
+  }
+
+  /**
+   * POJO class for printing out this command's output. This class is also used by the `terra config
+   * list` command, so it needs to be public.
+   */
+  public static class LoggingReturnValue {
+    // global logging context = log levels for file and stdout
+    public Logger.LogLevel consoleLoggingLevel;
+    public Logger.LogLevel fileLoggingLevel;
+
+    public LoggingReturnValue(
+        Logger.LogLevel consoleLoggingLevel, Logger.LogLevel fileLoggingLevel) {
+      this.consoleLoggingLevel = consoleLoggingLevel;
+      this.fileLoggingLevel = fileLoggingLevel;
+    }
+  }
 
-    System.out.println(
+  /** Print this command's output in text format. */
+  public static void printText(LoggingReturnValue returnValue) {
```

Comment:
consider making this a non-static member function of LoggingReturnValue since we're exposing it and it's associated strongly with that class. Or does that mess with jackson? Maybe can use `LoggingReturnValue::printText` for the `Consumer<LoggingReturnValue>`?

Reply:
I don't think it would mess with Jackson, but if it's a non-static function, then we don't need to pass the `LoggingReturnValue` instance as an argument. But that argument is required by the `Consumer` function that the `formatOption` method expects.

I could make it a static method the `LoggingReturnValue` class instead, but that's not really consistent with how I've defined them everywhere else, where the return value object definition is also tightly coupled to the method that prints it out. I don't have a strong opinion either way, but prefer consistency with the other command classes for now.